pageserver/src/page_service.rs

```diff
@@ -765,6 +765,25 @@ impl postgres_backend::Handler for PageServerHandler {
                 Some(result.elapsed.as_millis().to_string().as_bytes()),
             ]))?
             .write_message(&BeMessage::CommandComplete(b"SELECT 1"))?;
+        } else if query_string.starts_with("checkpoint ") {
```

Comment:
It's needed by the new test, that's why it's here. Yeah, it's kind of unrelated though. It's in a separate commit.

Reply:
Pushed this first commit with the test now. I'll wait for the CI to run with that before pushing the second commit, so that we get before and after numbers from the new performance test.